album.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Album</title>
+    <link type="text/css" rel="stylesheet" href="CSS/ini.css" />
+    <link type="text/css" rel="stylesheet" href="CSS/header.css" />
+    <link type="text/css" rel="stylesheet" href="CSS/container.css" />
+    <link type="text/css" rel="stylesheet" href="CSS/footer.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <header class="header">
+        <svg xmlns="http://www.w3.org/2000/svg" width="20" 
```

Comment:
you would not need this svg for the first PR. This image usually should be in a separate folder.